engine/src/flutter/shell/platform/android/platform_view_android.cc

```diff
@@ -366,6 +367,10 @@ std::unique_ptr<Surface> PlatformViewAndroid::CreateRenderingSurface() {
 // |PlatformView|
 std::shared_ptr<ExternalViewEmbedder>
 PlatformViewAndroid::CreateExternalViewEmbedder() {
+  if (android_use_new_platform_view_) {
```

Comment:
This is the thing we switch on to enable it.

Still need to wire up a switch and 2) expose the ability to query this in Java and then add some dart code to conditionally use the right HC platform view.